src/utils/source-search.js

```diff
@@ -58,8 +64,12 @@ function ignoreWhiteSpace(str) {
  * @memberof utils/source-search
  * @static
  */
-function searchOverlay(query) {
-  query = new RegExp(escapeRegExp(ignoreWhiteSpace(query)));
+function searchOverlay(query, regexMatch) {
+  if (regexMatch && query !== "") {
+    query = new RegExp(query);
```

Comment:
@bomsy I remember you wrote this token function recently, if you could give this a look over it'd be much appreciated as it currently makes the overlay fail for regex matched queries.